assets/stylesheets/mixins/_style.scss

```diff
@@ -15,3 +15,11 @@
   overflow-scrolling: touch;
   -webkit-overflow-scrolling: touch;
 }
+
+@mixin circle {
+  border-radius: 999999px;
+}
+
+@mixin rounded($rounded) {
```

Comment:
I guess if it auto-prefixes then yes, true.

unless you think `.circle` is handy


> On Nov 16, 2015, at 6:26 PM, Michael Dijkstra <notifications@github.com> wrote:
> 
> In assets/stylesheets/mixins/_style.scss <https://github.com/micdijkstra/stencil/pull/20#discussion_r45000984>:
> 
> > @@ -15,3 +15,11 @@
> >    overflow-scrolling: touch;
> >    -webkit-overflow-scrolling: touch;
> >  }
> > +
> > +@mixin circle {
> > +  border-radius: 999999px;
> > +}
> > +
> > +@mixin rounded($rounded) {
> It's less characters to write border-radius: 10px than @include rounded(10px)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/micdijkstra/stencil/pull/20/files#r45000984>.
>